src/repr/relation.rs

```diff
@@ -184,10 +184,10 @@ impl RelationDesc {
 
     pub fn add_cols<I, N>(&mut self, cols: I)
     where
-        I: IntoIterator<Item = (ColumnType, Option<N>)>,
+        I: IntoIterator<Item = (Option<N>, ColumnType)>,
         N: Into<ColumnName>,
     {
-        for (typ, name) in cols.into_iter() {
+        for (name, typ) in cols.into_iter() {
```

Comment:
I don't mind, but just curious why the order is changing

Reply:
I explained the rationale above in the initial description.